### PR TITLE
105-end screen refining

### PR DESCRIPTION
**Pull request by EngineeringDude123**
Refining of EndScreen credits and related items

### CODE REVIEW

**Comment by EngineeringDude123** [comment]
Press C in the startmenu to skip to the credits

File: dunwoody_disaster/assets/ComputerLabResized.png
Comment (chanson02) [inline dunwoody_disaster/assets/ComputerLabResized.png:1]: Please keep branches related to the PR.

File: dunwoody_disaster/dialogues/Ryan Rengo.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
   },
   "Mitch": {
     "protagonist_lines": [
-      "What is it this time?", "This is outrageous! It’s unfair! How can you teach the same basic skills class after class after class and charge full price?!"
+      "What is it this time?", "This is outrageous! It's unfair! How can you teach the same basic skills class after class after class and charge full price?!"
```
Comment (chanson02) [inline dunwoody_disaster/dialogues/Ryan Rengo.json]: I missed these, I only did the double quotes in my PR.

File: dunwoody_disaster/views/MapScreen.py [hunk 2]
```diff
@@ -119,8 +127,8 @@ def available_rooms(self) -> list:
 
     def pixmap(self) -> QPixmap:
         rooms = self.coordinates()
-        result = QPixmap(self.image).scaledToWidth(750)  # original size 1024x1024
-
+        # Scale the map image
+        result = QPixmap(self.image).scaledToWidth(1270)  # original size 1024x1024
```
Comment (chanson02) [inline dunwoody_disaster/views/MapScreen.py:131]: Changing this is going to throw off the coordinates. I should really make a function for automatically scaling.

File: dunwoody_disaster/views/defeatScreen.py [hunk 3]
```diff
@@ -36,7 +36,9 @@ def __init__(self, image: str = None):
 
         # UI Elements
         # Game over label
-        defeat_label = QLabel("You have been defeated")
+        defeat_label = QLabel(
+            "The weight of disappointment was too much for you to bear. Your morale was broken."
```
Comment (chanson02) [inline dunwoody_disaster/views/defeatScreen.py]: Love this change!

File: dunwoody_disaster/views/creditScreen.py [hunk 4]
```diff
@@ -1,75 +1,134 @@
...
+        painter.drawText(name_x, name_y, name)
+        painter.drawText(description_x, desc_y, description)
 
-        for line in self.text_lines:
-            text_width = painter.fontMetrics().horizontalAdvance(line)
-            x = (self.width() - text_width) / 2
-            painter.drawText(x, y, line)
-            y += self.line_spacing
+    def initSound(self):
+        pygame.mixer.init()
```
Comment (chanson02) [inline dunwoody_disaster/views/creditScreen.py:120]: Wasn't this already inited in `main.py`? I'm not sure how the pygame mixer works, is this going to cause problems? Do we need to make an AudioHandler class to control what audio is playing?

**Review by chanson02 — COMMENTED** [review]
I love these credits! They are super cool!

File: dunwoody_disaster/assets/ComputerLabResized.png
Comment (chanson02) [reply dunwoody_disaster/assets/ComputerLabResized.png:1]: I changed the source branch and these went away :+1:

File: dunwoody_disaster/views/MapScreen.py (hunk 2 above)
Comment (EngineeringDude123) [reply dunwoody_disaster/views/MapScreen.py:131]: Ah I see, do you have an idea on how we should do it then?

File: dunwoody_disaster/views/creditScreen.py (hunk 4 above)
Comment (EngineeringDude123) [reply dunwoody_disaster/views/creditScreen.py:120]: As of now I'm currently using the pygame mixer as a multiswitch tool. 

e.g; load this audio -> play this audio -> upon switch turn off this audio -> load that audio -> play that audio -> repeat where necessary.

File: dunwoody_disaster/views/MapScreen.py (hunk 2 above)
Comment (chanson02) [reply dunwoody_disaster/views/MapScreen.py:131]: There is a function somewhere that does the math to convert them. You just need to change the original and target to match what you actually expect.

**Pull request merged**